docs/source/actions/dismiss_reviews.rst

```diff
@@ -32,17 +32,26 @@ Options
        pull request is updated. If set to ``false``, nothing will be done. If
        set to a list, each item should be the GitHub login of a user whose
        review will be removed.
+       If set to ``from_requested_reviewers``, The requested reviewers list
+       will be used to get whose review will be removed.
    * - ``changes_requested``
      - Boolean or list of string
      - ``true``
      - If set to ``true``, all the reviews requesting changes will be removed
        when the pull request is updated. If set to ``false``, nothing will be
        done. If set to a list, each item should be the GitHub login of a user
        whose review will be removed.
+       If set to ``from_requested_reviewers``, The requested reviewers list
+       will be used to get whose review will be removed.
    * - ``message``
      - :ref:`data type template`
      - ``Pull request has been modified.``
      - The message to post when dismissing the review.
+   * - ``on``
+     - ``code-change`` or ``always``
```

Comment:
Maybe use `synchronize` rather than `code-change`?
I feel it's safer to stick to GitHub terminology just in case, even if we have to explain what it means.